tox.ini

```diff
@@ -0,0 +1,18 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
Fixed! Note that I couldn't seem to figure out a way to get Travis to use Python 3.4, but tox will install it and run from there, so you'll see Travis report that there is "no language set" but has a TOXENV=py34.